ui-configuration.graphqls

```diff
@@ -23,11 +23,27 @@ type DashboardConfiguration {
     configuration: String!
 }
 
+type MenuItem {
+    # Display name
+    name: String!
+    # Showing icon name
+    icon: String
+    # Link path
+    path: String!
+    # Activated menu should be listed on the menu,
+    # otherwise, it should stay in marketplace.
+    activate: Boolean!
```

Comment:
@Fine0830 The new menu style should include a marketplace on the first, which should be able to expand, or a new page as the marketplace.